application.html

```diff
@@ -182,6 +189,13 @@ <h1 class="white title">Apply to Hack Camp</h1>
                 </li>
               </ul>
             </div>
+            <div class="small-12 columns">
+              <label>
+                Can you briefly (1-3 sentences) tell us about something you've
+                coded before and how it worked?
+                <input name=" entry.159477422" type="text" />
```

Comment:
Can you remove the space in the name and have this field only shown if they selected "I've completed a complete project" or more experience?

@jonleung thoughts?